cpp/src/arrow/parquet/parquet-schema-test.cc

```diff
@@ -45,6 +45,7 @@ const auto INT64 = std::make_shared<Int64Type>();
 const auto FLOAT = std::make_shared<FloatType>();
 const auto DOUBLE = std::make_shared<DoubleType>();
 const auto UTF8 = std::make_shared<StringType>();
+const auto TIMESTAMP = std::make_shared<TimestampType>();
```

Comment:
oh, I actually did not see that. *ups*, will have to improve a bit on my code then.